app/models/etl/dates.rb

```diff
@@ -1,3 +1,5 @@
+#TODO do we need this class?
```

Comment:
Do you need this #TODO?

Reply:
This was just a reminder to check if this class is needed, It is only used by tests so probably should be deleted.

Reply:
Ok, cool. Let's just remove it then 👍